sklearn/linear_model/logistic.py

```diff
@@ -623,20 +623,20 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
         mask = (y == pos_class)
         y_bin = np.ones(y.shape, dtype=np.float64)
         y_bin[~mask] = -1.
+        # for compute_class_weight
+
+        if class_weight in ("auto", "balanced"):
```

Comment:
Is there any internal comment convention for things that are deprecated? ```# TODO:``` looks horrible but some kind of tag could be nice.